text/0000-docs-team.md

```diff
@@ -0,0 +1,107 @@
+- Feature Name: N/A
+- Start Date: 2016-07-21
+- RFC PR: (leave this empty)
+- Rust Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+Create a team responsible for documentation for the Rust project.
+
+# Motivation
+[motivation]: #motivation
+
+[RFC 1068] introduced a federated governance model for the Rust project. Several initial subteams were set up. There was a note
+after the [original subteam list] saying this:
+
+[RFC 1068]: https://github.com/rust-lang/rfcs/blob/master/text/1068-rust-governance.md
+[original subteam list]: https://github.com/rust-lang/rfcs/blob/master/text/1068-rust-governance.md#the-teams
+
+> In the long run, we will likely also want teams for documentation and for community events, but these can be spun up once there is a more clear need (and available resources).
+
+Now is the time for a documentation subteam.
+
+## Why documentation was left out
+
+Documentation was left out of the original list because it wasn't clear that there would be anyone but me on it. Furthermore,
+one of the original reasons for the subteams was to decide who gets counted amongst consensus for RFCs, but it was unclear
+how many documentation-related RFCs there would even be.
+
+## Chicken, meet egg
+
+However, RFCs are not only what subteams do. To quote the RFC:
+
+> * Shepherding RFCs for the subteam area. As always, that means (1) ensuring
+>   that stakeholders are aware of the RFC, (2) working to tease out various
+>   design tradeoffs and alternatives, and (3) helping build consensus.
+> * Accepting or rejecting RFCs in the subteam area.
+> * Setting policy on what changes in the subteam area require RFCs, and reviewing direct PRs for changes that do not require an RFC.
+> * Delegating reviewer rights for the subteam area. The ability to r+ is not limited to team members, and in fact earning r+ rights is a good stepping stone toward team membership. Each team should set reviewing policy, manage reviewing rights, and ensure that reviews take place in a timely manner. (Thanks to Nick Cameron for this suggestion.)
+
+The first two are about RFCs themselves, but the second two are more pertinent to documentation. In particular,
+deciding who gets `r+` rights is important. A lack of clarity in this area has been unfortuante, and has led to a
+chicken and egg situation: without a documentation team, it's unclear how to be more involved in working on Rust's
+documentation, but without people to be on the team, there's no reason to form a team. For this reason, I think
+a small initial team will break this logjam, and provide room for new contributors to grow.
+
+# Detailed design
+[design]: #detailed-design
+
+The Rust documentation team will be responsible for all of the things listed above. Specifically, they will pertain
+to these areas of the Rust project:
+
+* The standard library documentation
+* The book and other long-form docs
```

Comment:
I'm *assuming* that "book" here is referring to [TRPL](https://doc.rust-lang.org/book/) and not [rust-lang/book](https://github.com/rust-lang/book). Is that a correct assumption or do you see the docs team also contributing to the latter?